sqlglot/parser.py

```diff
@@ -893,7 +894,7 @@ def _parse_auto_increment(self):
         return self.expression(
             exp.AutoIncrementProperty,
             this=exp.Literal.string("AUTO_INCREMENT"),
-            value=self._parse_var() or self._parse_number(),
+            value=self._parse_number(),
```

Comment:
This was changed because I couldn't find any documented examples where parsing a VAR made sense. Probably needs to be double-checked, though.